README.md

```diff
@@ -1,63 +1,64 @@
 
-# Importance of Version Control Systems while coding
+# Importance of Version Control Systems while Coding
 Written by Aran Arora, Sahil Singh Rawat & Priya Peter Menezes
 
 ## Introduction to Version Control
-Version control is a system and practice for managing changes in a file or digital content. By using a set of version control practices and tools, one can revert to a previous state of a project and view or manipulate its content. These are crucial tools that ensure that play a big part in ensuring the quality of the final product.
+Version control is _\[both\]_ a system and practice for managing changes in a _\[centralized space that allows contributors to access, modify and contribute\]_ file or digital content _\[simultaneously\]_. By using a set of version control practices and tools, one can revert to a previous state of a project and view or manipulate its content. These are crucial tools that ensure that play a big part in ensuring the quality of the final product.
 
-Version control systems provide a centralized space for the project which allows contributors to access, modify and contribute simultaneously.
+~~Version control systems provide a centralized space for the project which allows contributors to access, modify and contribute simultaneously.~~
```

Comment:
In a properly written article it is quite rare to have a paragraph consisting of only a single sentence. Sometimes this is done for emphasis, but that doesn't seem to be the case here. Better if you incorporate the sentence into another part of your text, as I have done above.